src/helpers/transform.js

```diff
@@ -0,0 +1,115 @@
+const isInvalid = (input) => input
+  && !/UNKNOWN|Unknown|NONE|N\/A|Invalid code|Invalid category.+/g.test(input);
+
+const replaceAddr = (addr, txt = '') => `${addr
+  .toString()
+  .replace(/UNKNOWN|Unknown|County| Sub County|Invalid code|[.,]+/g, '')
```

Comment:
Please remove the leading whitespace before `Sub County`. It might mess up your string matching

Reply:
You also need to call `.trim()` at the end to ensure that any resulting white spaces are removed

Reply:
Removed